base_contact/i18n/de.po

```diff
@@ -0,0 +1,196 @@
+"Language-Team: German <kde-i18n-de@kde.org>\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=UTF-8\n"
+"Content-Transfer-Encoding: 8bit\n"
+"Plural-Forms: \n"
+"X-Generator: Lokalize 1.5\n"
+"Language: de\n"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "City"
+msgstr "Stadt"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "other position"
+msgstr "andere Position"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Contacts"
+msgstr "Kontakte"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid ""
+"To see personal information about this contact, please go to to the his "
+"person form:"
+msgstr ""
+"Um persönliche Auskunft zu diesem Kontakt zu erhalten, gehen Sie bitte auf "
+"die Seite persönlich:"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Street..."
+msgstr "Straße..."
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "State"
+msgstr "Bundesland"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "at"
+msgstr "bei"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Tags..."
+msgstr "Schlagworte..."
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Other Positions"
+msgstr "Andere Positionen"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Phone:"
+msgstr "Telefon:"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Company"
+msgstr "Unternehmen"
+
+#. module: base_contact
+#: field:res.partner,contact_id:0
+msgid "Main Contact"
+msgstr "Hauptkontakt"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Fax:"
+msgstr "Fax:"
+
+#. module: base_contact
+#: code:addons/base_contact/base_contact.py:31
+#, python-format
+msgid "Standalone Contact"
+msgstr "Alleinstehender Kontakt"
+
+#. module: base_contact
+#: help:res.partner,nationality_id:0
+msgid "Nationality."
+msgstr "Nationalität."
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Address"
+msgstr "Anschrift"
+
+#. module: base_contact
+#: field:res.partner,nationality_id:0
+msgid "Nationality"
+msgstr "Nationalität"
+
+#. module: base_contact
+#: code:addons/base_contact/base_contact.py:32
+#, python-format
+msgid "Attached to existing Contact"
+msgstr "Mit bestehendem Kontakt verknüpft"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "ZIP"
+msgstr "PLZ"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Country"
+msgstr "Land"
+
+#. module: base_contact
+#: field:res.partner,birthdate_date:0
+msgid "Birthdate"
+msgstr "Geburtsdatum"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Person"
+msgstr "Person"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Contact"
+msgstr "Kontakt"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "Mobile:"
+msgstr "Mobil:"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "All partner positions"
+msgstr "Alle Positionen des Partners"
+
+#. module: base_contact
+#: view:res.partner:0
+msgid "{'invisible': [('contact_type','=','attached')]}"
```

Comment:
Better remove this line that can be conflictive (it's a bug in the translation export)